net/FlatBuffers/FlatBufferBuilder.cs

```diff
@@ -90,6 +90,10 @@ public void Clear()
             _objectStart = 0;
             _numVtables = 0;
             _vectorNumElems = 0;
+            if (_sharedStringMap != null)
+            {
+                _sharedStringMap.Clear();
```

Comment:
Should we just null the Map and let the GC handle the memory? Clearing just removes items, but not capacity, so this would leave some memory on the tabel.

Reply:
Makes sense, will change the code to null it instead.

Reply:
I am fine with either approach, clearing or nulling.